evaluation/eval.py

```diff
@@ -37,65 +32,63 @@ class EvaluationArguments:
         default=None,
         metadata={"help": "Pretrained tokenizer name or path if not the same as model_name."}
     )
-    output_dir: Optional[str] = field(
+    output_dir: str = field(
         default="outputs",
         metadata={"help": "Directory for saving evaluation outputs."}
     )
-    random_seed: Optional[int] = field(
+    tag: Optional[str] = field(
+        default=None,
+        metadata={"help": "Identifier for the evaluation run."}
+    )
+    random_seed: int = field(
         default=24,
         metadata={"help": "Customized random seed"}
     )
     eval_tasks: Optional[List[str]] = field(
         default=None,
         metadata={"help": "A list of tasks to run the evaluation on, e.g. tydiqa_secondary"}
     )
-
-
-def main():
-    # parse arguments
-    parser = HfArgumentParser(EvaluationArguments)
-    eval_args, = parser.parse_args_into_dataclasses()
```

Comment:
That makes a lot of sense. I also learned just yesterday that putting parsers in `if __name__ == "__main__"` is not good practice since it introduces global variables (the parser). I moved the block back into `main()` as per your suggestion. Thanks!